packages/react-components/react-slider/library/src/components/Slider/useSliderState.tsx

```diff
@@ -4,12 +4,40 @@ import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts
 import { sliderCSSVars } from './useSliderStyles.styles';
 import type { SliderState, SliderProps } from './Slider.types';
 
-const { sliderStepsPercentVar, sliderProgressVar, sliderDirectionVar } = sliderCSSVars;
+const {
+  sliderStepsPercentVar,
+  sliderProgressVar,
+  sliderProgressValueVar,
+  sliderDirectionVar,
+  sliderStepsColorVar,
+  sliderStepsGradientVar,
+} = sliderCSSVars;
 
 const getPercent = (value: number, min: number, max: number) => {
   return max === min ? 0 : ((value - min) / (max - min)) * 100;
 };
 
+/**
+ * Generates a linear gradient CSS value representing the steps of a slider.
+ *
+ * @param stepSize - The step size for each gradient stop.
+ * @returns The linear gradient CSS value.
+ */
+const getStepsGradient = (stepSize: number) => {
```

Comment:
we are passing here `stepsPercent` value, which doesn't convey the meaning of `stepSize`, can we improve this logic/naming/docs ?

Reply:
Updated PR, this is no longer relevant